package.json

```diff
@@ -47,19 +47,19 @@
     "react-dom": "^16.4.2",
     "react-draggable": "^3.0.5",
     "react-helmet": "^5.1.3",
-    "react-highlight-words": "^0.13.0",
+    "react-highlight-words": "^0.14.0",
     "react-images": "^0.5.19",
     "react-scroll": "^1.7.10",
     "react-slick": "^0.23.1",
     "reactstrap": "^6.4.0",
     "sass-loader": "^7.1.0",
-    "scrivito": "^1.2.0",
+    "scrivito": "^1.3.0-rc1",
```

Comment:
In the example app we are always using the latest stable version of `scrivito`. `1.3.0-rc1` is a release candidate and therefore not considered stable. Let's use version `1.2.0` until `1.3.0` (final) is out.